js/app.js

```diff
@@ -1,283 +1,109 @@
 'use strict';
 
-var total = 0;
-
 function randNum(min, max) {
   return Math.floor(Math.random() * (max - min + 1)) + min;
 }
-
-var firstAndPike = {
-  name: 'First and Pike',
-  minCust: 23,
-  maxCust: 65,
-  avgCookieSales: 6.3,
-  cookiesPerHour: [],
-  getCookiesPerHour: function() {
-    for (var i = 0; i < 15; i++) {
+var storename = [];
+function CookieStore (name, minCust, maxCust, avgCookieSales, cookiesPerHour) {
+  this.name = name;
+  this.minCust = minCust;
+  this.maxCust = maxCust;
+  this.avgCookieSales = avgCookieSales;
+  this.cookiesPerHour = cookiesPerHour;
+  this.getCookiesPerHour = function() {
+    for (var i = 0; i < 14; i++) {
```

Comment:
It's good practice to not hard code how many time a for loop iterates. This should loop through the length of perhaps an array so in case you change how many stores or store hours your code still works.